static/global.css

```diff
@@ -1,21 +1,62 @@
+/*
 @font-face {
-  font-family: 'Open Sans Light';
-  src: url('/font/OpenSans-Light.ttf') format('truetype');
+	font-family: 'Open Sans Light';
+	src: url('/font/OpenSans-Light.ttf') format('truetype');
 }
 @font-face {
-  font-family: 'Open Sans Regular';
-  src: url('/font/OpenSans-Regular.ttf') format('truetype');
+	font-family: 'Open Sans Regular';
+	src: url('/font/OpenSans-Regular.ttf') format('truetype');
 }
 @font-face {
-  font-family: 'Open Sans Semibold';
-  src: url('/font/OpenSans-Semibold.ttf') format('truetype');
+	font-family: 'Open Sans Semibold';
+	src: url('/font/OpenSans-Semibold.ttf') format('truetype');
```

Comment:
I'd rename `/font/unused/` `/font/unused-OpenSans/*` and I'd move `/font/OpenSans*` to `/font/unused-OpenSans/*`